### ISSUE TITLE
cmd/trace: crash because of deprecated HTML Imports in web view

### ISSUE THREAD

**Issue by mmathys** [golang/go#38054]
### What version of Go are you using (`go version`)?

<pre>
go version go1.14 darwin/amd64
</pre>

### Does this issue reproduce with the latest release?

Yes

### What operating system and processor architecture are you using (`go env`)?

Chrome 80.0.3987.132 MacOS

<details><summary><code>go env</code> Output</summary><br><pre>
$ go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/max/Library/Caches/go-build"
GOENV="/Users/max/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/max/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/Cellar/go/1.14/libexec"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/Cellar/go/1.14/libexec/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/rg/bvn53d7j0wbfnxkw_fktlsq40000gn/T/go-build837065189=/tmp/go-build -gno-record-gcc-switches -fno-common"
</pre></details>

### What did you do?

Run the go trace tool:`go tool trace -http=':8081' trace.out` on any trace. Then, navigate to to **View Trace**. This page is using a HTML imports, [which have been deprecated in Chrome M80](https://developers.google.com/web/updates/2019/07/web-components-time-to-upgrade).

### What did you expect to see?

The trace tool.

### What did you see instead?

An empty page and an error in the console:

```
HTML Imports is deprecated and has now been removed as of M80. See https://www.chromestatus.com/features/5144752345317376 and https://developers.google.com/web/updates/2019/07/web-components-time-to-upgrade for more details.
Uncaught ReferenceError: tr is not defined
    at onResult (trace:59)
    at trace:43
```

**Comment by seankhliao** [comment]
should be fixed in 14.1 #37343

**Comment by andybons** [comment]
Duplicate of #34374
